src/Services/Identity/Identity.Infrastracture/Implementations/UserRepository.cs

```diff
@@ -48,7 +48,7 @@ public async Task<string> GetUserRoleAsync(string userId)
     {
         var user = await _userManager.FindByIdAsync(userId);
 
-        return (await _userManager.GetRolesAsync(user)).Single();
+        return (await _userManager.GetRolesAsync(user!)).Single();
```

Comment:
SingleOrDefault

Reply:
return null if user not found

Reply:
Is there any situation when I should use just Single or should I always use SingleOrDefault?